java/code/src/com/redhat/rhn/common/conf/Config.java

```diff
@@ -483,7 +483,7 @@ public void parseFiles() {
                 props.load(new StringReader(configString.replace("\\", "\\\\")));
             }
             catch (IOException e) {
-                logger.error("Could not parse file {}", curr, e);
+                logger.error("Could not parse file {}", curr);
```

Comment:
Wouldn't it be better to show the error?
```suggestion
                logger.error("Could not parse file {}: {}", curr, e);
```

Reply:
Right, I removed the error from the log, because in a local testing context this error is expected. So I wanted to hide the long stacktrace from the console for each local test run from the IDE. Maybe we could use `logger.error("Could not parse file {}: {}", curr, e.getMessage());` at least, will try it out.

Reply:
Removed this part of the patch for now, I also have not seen the respective stacktrace anymore recently.